app/src/main/java/org/greenstand/android/TreeTracker/utilities/CameraHelper.kt

```diff
@@ -14,14 +14,18 @@ object CameraHelper {
     fun takePictureForResult(fragment: Fragment, selfie: Boolean) {
         if (ActivityCompat.checkSelfPermission(
                 fragment.requireContext(),
-                Manifest.permission.CAMERA) != PackageManager.PERMISSION_GRANTED ||
+                Manifest.permission.CAMERA
```

Comment:
Was al this just reformatting?

Reply:
Yeah I'm not sure what happened here to be honest. I'll revert the changes in here